pkg/operator/configobserver/network/observe_network.go

```diff
@@ -0,0 +1,111 @@
+package network
+
+import (
+	"fmt"
+
+	"github.com/ghodss/yaml"
+	"github.com/golang/glog"
+	"github.com/openshift/library-go/pkg/operator/events"
+	"k8s.io/client-go/listers/core/v1"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+const (
+	clusterConfigNamespace = "kube-system"
+	clusterConfigName      = "cluster-config-v1"
+)
+
+// GetClusterCIDRs reads the cluster CIDRs from the install-config ConfigMap in the cluster.
+func GetClusterCIDRs(lister v1.ConfigMapLister, recorder events.Recorder) ([]string, error) {
+	clusterConfig, err := lister.ConfigMaps(clusterConfigNamespace).Get(clusterConfigName)
+	if errors.IsNotFound(err) {
+		recorder.Warningf("ObserveClusterCIDRFailed", "Required %s/%s config map not found", clusterConfigNamespace, clusterConfigName)
+		glog.Warning("configmap/cluster-config-v1.kube-system: not found")
```

Comment:
@deads2k @sttts every event also generate a glog message when created... I wonder if we can deduplicate these to get cleaner logs?

Reply:
a configobserver/log package. Same for events.

Reply:
> @deads2k @sttts every event also generate a glog message when created... I wonder if we can deduplicate these to get cleaner logs?

you mean just delete the glog?